README.md

```diff
@@ -118,6 +118,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="http://dylancorriveau.ca"><img src="https://avatars1.githubusercontent.com/u/1254251?v=4" width="100px;" alt=""/><br /><sub><b>Dylan Corriveau</b></sub></a><br /><a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=dcorriveau" title="Code">💻</a> <a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=dcorriveau" title="Documentation">📖</a></td>
     <td align="center"><a href="https://github.com/vybhav72954"><img src="https://avatars3.githubusercontent.com/u/49750343?v=4" width="100px;" alt=""/><br /><sub><b>vybhav72954</b></sub></a><br /><a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=vybhav72954" title="Code">💻</a> <a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=vybhav72954" title="Documentation">📖</a> <a href="#maintenance-vybhav72954" title="Maintenance">🚧</a></td>
     <td align="center"><a href="https://www.linkedin.com/in/sharvari-raut-a62a99171"><img src="https://avatars1.githubusercontent.com/u/57194200?v=4" width="100px;" alt=""/><br /><sub><b>Sharvari Raut</b></sub></a><br /><a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=sharur7" title="Code">💻</a> <a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=sharur7" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/pankaj892"><img src="https://avatars3.githubusercontent.com/u/31444506?v=4" width="100px;" alt=""/><br/><sub><b>Pankaj Biradar</b></sub></a><br /><a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=pankaj892" title="Documentation">📖 💻 👨‍💻</a></td>
```

Comment:
You don't need to make this change manually. The All Contributors Bot does this for us, automatically.